src/govuk/helpers/_links.scss

```
@@ -240,13 +245,13 @@
   }
 }
 
-/// Text style link mixin
+/// Text link styles
 ///
-/// Overrides the colour of links to match the text colour. Generally used by
-/// navigation components, such as breadcrumbs or the back link.
+/// Makes links use the primary text colour, in all states. Use for navigation
```

```suggestion
/// Makes links use the primary text colour, in all states. Use this mixin for navigation
```
To keep consistent with updates further up the docs, and to help it flow more nicely.